src/app/component/ProductCompare/ProductCompare.style.scss

```diff
@@ -0,0 +1,147 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+:root {
+    --product-compare-area-width: var(--content-wrapper-width);
+    --product-compare-sidebar-width: 200px;
+    --product-compare-item-count: 3.2;
+
+    @include mobile {
+        --product-compare-area-width: 100vw;
+        --product-compare-sidebar-width: 0px;
+        --product-compare-item-count: 2.1;
+    }
```

Comment:
Consider declaring mobile overrides after rest of the variables.

Reply:
Fixed